account_invoice_triple_discount/README.rst

```diff
@@ -6,7 +6,7 @@
 Account Invoice Triple Discount
 ===============================
 
-This module allows to have three successive discounts on every invoice line.
+This module allows to have three consecutive discounts on each invoice line.
```

Comment:
Well, I'd say _successive_ is the proper term here (https://wordpandit.com/successive-percentage-change/).